sample-compose/app/src/androidTest/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreenTest.kt

```diff
@@ -0,0 +1,86 @@
+package co.nimblehq.sample.compose.ui.screens.home
+
+import androidx.activity.compose.setContent
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.ComposeContentTestRule
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.GrantPermissionRule
+import co.nimblehq.sample.compose.domain.model.Model
+import co.nimblehq.sample.compose.domain.usecase.UseCase
+import co.nimblehq.sample.compose.test.CoroutineTestRule
+import co.nimblehq.sample.compose.ui.AppDestination
+import co.nimblehq.sample.compose.ui.screens.MainActivity
+import co.nimblehq.sample.compose.ui.theme.ComposeTheme
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.flowOf
+import org.junit.*
+import org.junit.Assert.assertEquals
+import org.junit.runner.RunWith
+
+@ExperimentalTestApi
```

Comment:
Do we need this?
```suggestion
```

Reply:
@Wadeewee it's redundant, good point 👍

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/384/commits/bda2a6a742ae459b455588e700ca4d387716ec29